sn_protocol/src/messages/response.rs

```diff
@@ -44,18 +43,12 @@ pub enum QueryResponse {
     /// [`GetReplicatedData`]: crate::messages::Query::GetReplicatedData
     GetReplicatedData(Result<(NetworkAddress, ReplicatedData)>),
     //
-    // ===== Register Data =====
+    // ===== Register =====
     //
-    /// Response to [`RegisterQuery::Get`].
-    GetRegister(Result<Register>),
-    /// Response to [`RegisterQuery::GetEntry`].
-    GetRegisterEntry(Result<Entry>),
-    /// Response to [`RegisterQuery::GetOwner`].
-    GetRegisterOwner(Result<User>),
-    /// Response to [`RegisterQuery::Read`].
-    ReadRegister(Result<BTreeSet<(EntryHash, Entry)>>),
-    /// Response to [`RegisterQuery::GetPermissions`].
-    GetRegisterPermissions(Result<Permissions>),
+    /// Response to [`GetRegister`]
+    ///
+    /// [`GetRegister`]: crate::messages::Query::GetRegister
+    GetRegister(Result<SignedRegister>),
```

Comment:
All this was un-secure AND unused by the client, replaced it with a simpler and secure API.